Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset

```diff
@@ -220,7 +220,7 @@ MonoBehaviour:
   sourceBankPath: FMOD Build Files/Desktop
   SourceBankPathUnformatted: 
   BankRefreshCooldown: 5
-  ShowBankRefreshWindow: 1
```

Comment:
Do we want this to show?

Reply:
Honestly, it's up to the user.  I'm tempted to add this to the gitignore for our purposes

Reply:
I'll revert for now.